src/pages/studyView/styles.module.scss

```diff
@@ -117,3 +117,18 @@
         color: #fff !important;
     }
 }
+
+.tooltip {
+    max-width: 300px;
+}
+
+$wide: 10px;
```

Comment:
seems kind of strange.  investigate where this comes from

Reply:
This code comes from _src/shared/components/PillTag/styles.module.scss_. So, I did not create this code. What shall we do with it?